qemu/tests/block_vhost_vdpa_test.py

```diff
@@ -0,0 +1,136 @@
+"""VDPA blk vhost vdpa test"""
+
+from avocado.utils import process
+
+from provider.block_devices_plug import BlockDevicesPlug
+from provider.vdpa_sim_blk_utils import VhostVdpaSimulatorTest
+from virttest import env_process, utils_disk, utils_misc
+from virttest.utils_misc import get_linux_drive_path
+from virttest.utils_windows.drive import get_disk_props_by_serial_number
+
+
+def run(test, params, env):
+    """
+    VDPA blk vhost vdpa test. It tests multiple scenarios:
+    multi-disks,hotplug-unplug,and blockdev options
+    Multi-disks steps:
+        1) Setup VDPA simulator ENV and create vhost-vdpa devices.
+        2) Boot VM with multi vdpa-blk disks
+        3) Verify disk in guest
+        4) Destroy VM
+        5) Destroy vhost-vdpa devices on host
+        7. Destroy VDPA simulator ENV
+    Hotplug-unplug test steps:
+        1) Setup VDPA simulator ENV and create vhost-vdpa devices.
+        2) Boot VM
+        4) Hotplug the vhost-vdpa disks
```

Comment:
It should be 3 instead of 4. And please also update the next lines (line 27 - 31). Thanks.

Reply:
updated